resources/carrier/en/380.txt

```diff
@@ -13,19 +13,26 @@
 # limitations under the License.
 
 # Data taken from:
-# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0004PDFE.pdf
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0005PDFE.pdf
 
 # Telesystems of Ukraine is commonly known as PEOPLEnet.
 
 38050|Vodafone
-38063|Life
+38063|Lifecell
 38066|Vodafone
 38067|Kyivstar
 38068|Kyivstar
-38073|Life
+38073|Lifecell
+380891|Datagroup
+380892|Ukrtelecom
+380893|T.R. Communicatio
+380894|Atlantic Telecom
+380895|Lincom-300
+380897|Kyivstar
+380899|Velton Telecom
```

Comment:
All the prefixes starting with 89 are more of Voip numbers than Mobile. So we do not add carrier details for them. So please remove them

Reply:
Fixed. But why?